x-pack/plugins/security_solution/server/endpoint/services/artifacts/manifest_manager/manifest_manager.ts

```diff
@@ -686,7 +686,7 @@ export class ManifestManager {
       },
     });
 
-    for await (const policies of this.fetchAllPolicies()) {
+    for await (const policies of await this.fetchAllPolicies()) {
```

Comment:
This syntax looks strange to me, are you sure this works?

Reply:
Yes it works, we now have some similar syntaxes in Fleet code too